pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -185,3 +186,11 @@ def tensorfromscalar(x):
         return torch.as_tensor(x)
 
     return tensorfromscalar
+
+
+@pytorch_funcify.register(Split)
+def pytorch_funcify_Split(op, node, **kwargs):
+    def inner_fn(x, dim, split_amounts):
```

Comment:
A more descriptive name is nice because it shows in the compiled function

Reply:
Yea your totally right